packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/glue/start-job-run.ts

```diff
@@ -167,35 +175,52 @@ export enum WorkerType {
   /**
    * Each worker provides 4 vCPU, 16 GB of memory and a 50GB disk, and 2 executors per worker.
    */
-  STANDARD ='Standard',
+  STANDARD = 'Standard',
 
   /**
    * Each worker maps to 1 DPU (4 vCPU, 16 GB of memory, 64 GB disk), and provides 1 executor per worker. Suitable for memory-intensive jobs.
    */
-  G_1X ='G.1X',
+  G_1X = 'G.1X',
 
   /**
    * Each worker maps to 2 DPU (8 vCPU, 32 GB of memory, 128 GB disk), and provides 1 executor per worker. Suitable for memory-intensive jobs.
    */
-  G_2X ='G.2X',
+  G_2X = 'G.2X',
 
   /**
    * Each worker maps to 4 DPU (16 vCPU, 64 GB of memory, 256 GB disk), and provides 1 executor per worker. We recommend this worker type for jobs whose workloads contain your most demanding transforms, aggregations, joins, and queries. This worker type is available only for AWS Glue version 3.0 or later jobs.
    */
-  G_4X ='G.4X',
+  G_4X = 'G.4X',
 
   /**
    * Each worker maps to 8 DPU (32 vCPU, 128 GB of memory, 512 GB disk), and provides 1 executor per worker. We recommend this worker type for jobs whose workloads contain your most demanding transforms, aggregations, joins, and queries. This worker type is available only for AWS Glue version 3.0 or later jobs.
    */
-  G_8X ='G.8X',
+  G_8X = 'G.8X',
 
   /**
    * Each worker maps to 0.25 DPU (2 vCPU, 4 GB of memory, 64 GB disk), and provides 1 executor per worker. Suitable for low volume streaming jobs.
    */
-  G_025X ='G.025X',
+  G_025X = 'G.025X',
```

Comment:
Can you please move `G_025X` between `STANDARD` and `G_1X`?
It seems a bit out of place here.

Reply:
@lpizzinidev 
Thanks. I've changed the order.